src/scripts/hunt-filter/engine.test.ts

```diff
@@ -0,0 +1,199 @@
+import {IntakeRejectionEngine} from './engine';
+import {ApiResponse, User} from '../types/hg';
+import {IntakeMessage} from '../types/mhct';
+import {Logger} from '../util/logger';
+jest.mock('../util/logger')
+
+// Mock logger won't actually call console.log
+const logger = new Logger();
+const engine = new IntakeRejectionEngine(logger);
+
+describe('validateResponse', () => {
```

Comment:
I don't see it in dist. Webpack leaves it out since it is never used in any imports in the entries.

I did have it in a `./tests/scripts/hunt-filter` (ie mirror), but then you get really verbose import statements
```
import {blah} from '../../../src/scripts/hunt-filter/blah'
```
instead of 
```
import {blah} from './blah'
```

Reply:
i'm ok with long paths, i agree they should be in separate folder. Maybe some {root_path}/src/... variable?

Reply:
also, these tests are used inside the source, no? i'm confused. I thought we were doing tests to check if pre/post object matches, not tests in the sense that we test if parts of extension works. If i'm correct, then this should be part of the src, no?

the other tests like unit tests, cucumber, etc, should be in separate folder in my experience.

Reply:
I will move to mirror folder and create a var to shorten things up.

These tests are not part of source. They are exclusively unit testing the methods + rules.
Each test
1. Grabs a valid pre + post (ApiReponse | User | IntakeMessage)
2. Modifies a properties to make it invalid since pre and post differ
3. Runs the appropriate `validate` method.
4. Generally asserts `false`. But in the case of testing exemptions, it assert `true`.